tests/testthat/test_learner_arima.R

```diff
@@ -0,0 +1,60 @@
+context("LearnerArima")
+
+# test_that("autotest", {
+#   learner = LearnerRegrForecastArima$new()
+#   expect_learner(learner)
+#   result = run_autotest(learner)
+#   expect_true(result, info = result$error)
+# })
+
+test_that("Basic Tests", {
+  task = tsk("airpassengers")
+  learner = lrn("forecast.arima")
+  learner$train(task, 1:20)
+  fitted_values = learner$fitted_values(row_ids = 5:10)
+  expect_data_table(fitted_values, nrows = 6, ncols = length(task$target_names), types = "numeric")
+  expect_prediction(learner$predict(task, 21:30))
+
+  resampling = ResamplingCustom$new()
+  resampling$instantiate(task, train_sets = list(1:100), test_sets = list(101:144))
```

Comment:
I wouldnt test the Resample Result but focus more on the Prediction object itself (if the learner API works then the ResampleResult should work as well)

Reply:
e.g. check that the fitted values are correctly contained in the prediction.

Reply:
Also expect_learner should be run I guess